README.md

```diff
@@ -176,16 +176,16 @@ worksheet.batch_update([{
 
 ## [Documentation](https://gspread.readthedocs.io/en/latest/)
```

Comment:
I think as we are currently updating that part we can change something too here:

- I always get confused when I see a link on the header of a markdown file.
- We could leave the header with `Documentation`
- and move the actual link to a basic paragraph bellow like "Documentation can be found here" with the link on "here"

What do you think ?